src/harness/certs/README.md

```diff
@@ -24,6 +24,13 @@ https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/
                    sas_expired                                               domain_proxy_expired
                                                                              domain_proxy_inapplicable
 
+                  -------------root_ca --------------------------------------                
+                 /                   \         				      \ 
+                /                     \         			       \
+       blacklisted_sas_ca         blacklisted_cbsd_ca   		blacklisted_proxy_ca
+                \                        \                                   \
+                 \                        \                                   \
+            sas_revoked_by_ca         client_revoked_by_ca              domain_proxy_revoked_by_ca
```

Comment:
I think `sas_cert_from_revoked_ca` is more accurate. The cert itself isn't revoked, but its issuer is.

Reply:
leaf   certificate names are updated correctly to give clear indication signed by   revoked CA.